packages/fxa-content-server/tests/functional/settings_v2/external_links.js

```diff
@@ -42,4 +50,23 @@ describe('external links', () => {
       'https://www.mozilla.org/en-US/about/legal/terms/services/'
     );
   });
+
+  it('renders Newsletters link in navigation when we are subscribed to a product', async () => {
+    if (process.env.CIRCLECI === 'true' && !process.env.SUBHUB_STRIPE_APIKEY) {
+      this.skip('missing Stripe API key in CircleCI run');
+    }
+
+    const email = createEmail();
+    await subscribeAndSigninToRp(email);
+    await openPage(SETTINGS_V2_URL, selectors.SETTINGS_V2.HEADER);
+
+    // test
+    await testElementExists(
+      selectors.SETTINGS_V2.NAVIGATION.SUBSCRIPTIONS_LINK
+    );
+    await testHrefEquals(
+      selectors.SETTINGS_V2.NAVIGATION.SUBSCRIPTIONS_LINK,
+      `/subscriptions`
+    );
```

Comment:
Neither of these are testing for newsletter links?

Reply:
Updated the description

Reply:
The PR title/commit message summary is talking about a newsletter link test though?  Is this for a different issue?

Reply:
I mistakenly called it newsletter test instead of the subscriptions test, will update the commit message